R/helpers.R

```diff
@@ -90,6 +90,26 @@ use_dependency <- function(package, type, min_version = NULL) {
   invisible()
 }
 
+use_system_requirement <- function(requirement) {
+  stopifnot(is_string(requirement))
+  existing_requirements <- desc::desc_get_field("SystemRequirements", default = character(), file = proj_get())
+  existing_requirements <- utils::head(strsplit(existing_requirements, ", ?"), n = 1)
```

Comment:
I used head here rather than `[[1]]` to deal with the 0 length case, which is common if the package does not already have a SystemRequirements field. e.g.

``` r
utils::head(character(), n = 1)
#> character(0)
character()[[1]]
#> Error in character()[[1]]: subscript out of bounds
```

<sup>Created on 2020-07-02 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>